rpc_jobs/rpc_designate.yml

```diff
@@ -1,5 +1,5 @@
  - project:
-    name: "rpc-designate-pre-merge"
+    name: "rpc-designate-master-xenial-pre-merge"
```

Comment:
Hey @leader716, I'd leave this change out as you may end up adding more images down the line.